### PR TITLE
Debug Vplan - Updates & Link-to-cov

### PR DESCRIPTION
**Pull request by silabs-robin**
This PR updates the debug vplan.

It is the one we all edited in parallel.
It contains both added link-to-coverage, and it was also updated along with updates to the debug spec some time ago.

### CODE REVIEW

File: cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv
Comment (silabs-robin) [inline cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv:1]: > I saw I had to change placement of link to coverage, add some assertions and remove some outdated ones. So maybe uplaod a new version, hehe... And I am thinking that this is probably something to address now/soon:
> -- @silabs-krdosvik 

This is fixed with your latest changes, right?

Comment (silabs-robin) [inline cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv:1]: > And, this point ["dcsr" writability] dont seem to be covered
> -- @silabs-krdosvik 

I'm not sure what to do about that... yet...

**Review by silabs-robin — COMMENTED** [review]
Thanks for the review @silabs-krdosvik .

File: cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv
Comment (silabs-krdosvik) [reply cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv:1]: The minor changes is fixed with the latest changes, yes.

Comment (silabs-krdosvik) [reply cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv:1]: I guess I would just let it be, and remember to do it later?? (Add an issue maybe?)

Comment (silabs-robin) [reply cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv:1]: Agreed. Let's check in all of these updates, then fill in the missing items in a next iteration.

Comment (silabs-robin) [reply cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.csv:1]: Then I'm resolving this conversation.

**Review by silabs-hfegran — APPROVED** [review]

**Pull request merged**